ngraph/core/src/op/convert.cpp

```diff
@@ -104,6 +104,7 @@ namespace convert
             TYPE_OUT_CASE(bf16, arg, out);
             TYPE_OUT_CASE(f16, arg, out);
             TYPE_OUT_CASE(f32, arg, out);
+            TYPE_OUT_CASE(f64, arg, out);
```

Comment:
Inference Engine doesn't support `f64` data type. Why do we need to support it?

Reply:
This is just for constant folding. These models have Convert (i64->fp64) -> Sqrt -> Convert (fp64 -> i64).

Reply:
This constant folding affects the binary size. It is why I asked about this data type